src/common/eeprom.cpp

```diff
@@ -24,6 +21,23 @@ static const constexpr uint16_t EEVAR_FLG_READONLY = 0x0001; // variable is read
 // measure time needed to update crc
 //#define EEPROM_MEASURE_CRC_TIME
 
+#if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)
+enum {
+    MAX_SHEETS = 8,
+    EEPROM_SHEET_SIZEOF = 12
+};
+
+typedef struct
+{
+    char name[MAX_SHEET_NAME_LENGTH]; //!< Can be null terminated, doesn't need to be null terminated
+    float z_offset;                   //!< Z_BABYSTEP_MIN .. Z_BABYSTEP_MAX = Z_BABYSTEP_MIN*2/1000 [mm] .. Z_BABYSTEP_MAX*2/1000 [mm]
+} Sheet;
+
+    #ifdef __cplusplus
```

Comment:
`#ifdef __cplusplus` is probably redundant, eeprom.cpp should be already compiled by a c++ compiler